environment.yml

```diff
@@ -10,7 +10,7 @@ dependencies:
   - ipython
   - notebook
   - ipykernel
-  - pytorch
+  - pytorch>=1.8
```

Comment:
Somehow something installed a previous dependency and we couldn't find previously unexisting function on torch (``make deterministic``)